playbooks/files/rds/pg2/db_methods.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import uuid
+from contextlib import closing
+
+import psycopg2
+import yaml
+from psycopg2 import sql, Error, OperationalError
+
+from arg_parser import get_connection_dict, get_source
+
+
+def _logging_configuration():
+    """Basic configuration for logging"""
+    return logging.basicConfig(
+        filename = 'tmp/rds/rds_logs.log',
+        filemode = 'w',
+        level = logging.DEBUG,
+        format='%(levelname)s:%(asctime)s:%(message)s')
+
+
+def _execute_sql(sql_query: str) -> list:
+    """Execute sql query"""
+    connection_dict = get_connection_dict()
+    res = list()
+    try:
+        with closing(psycopg2.connect(**connection_dict)) as connection:
+            try:
+                with connection.cursor() as cursor:
+                    cursor.execute(sql_query)
+                    if cursor.description is not None:
+                        res = cursor.fetchall()
+            except OperationalError as err:
+                logging.error('Exception occured when try to execute SQL', exc_info = True)
+            connection.commit()
+    except Error as e:
+        logging.error('Connection error occured', exc_info = True)
+    return res
+
+
+def create_table(schema_name: str, table_name: str, *columns):
+    """Create table in database if it not exists"""
+    model_query = sql.SQL(
+        "create table if not exists {}.{}(id bigint generated by default as identity primary key,{} text);")
+    create_table_query = model_query.format(
+                                            sql.Identifier(schema_name),
+                                            sql.Identifier(table_name),
+                                            sql.SQL(', ').join(map(sql.Identifier, columns))
+                                           )
+    _execute_sql(create_table_query)
+    logging.info('Table was created (if it not existed before)')
+
+
+def generate_random_values_and_insert_into_table(schema_name: str, table_name: str, range_start: int, range_stop: int, *columns):
```

Comment:
`too_long_func_name`
what about `fill_table`?

Reply:
Change to fill_table_by_random_values